R/teal_data_module-eval_code.R

```diff
@@ -24,6 +22,17 @@ NULL
 #'   }
 #' )
 #' eval_code(tdm, "IRIS <- subset(IRIS, Species == 'virginica')")
+#'
```

Comment:
Given only 1 of the S4 functions is being shown, how about keeping it without the "S4 method..."? (just like teal.code::eval_code)

```suggestion
#'
#' @usage eval_code(object, code)
```

Reply:
I tried!

Thank you!

Reply:
Oh no. I generated it, but didn't run check. Too bad :-\